src/iota/view.rs

```diff
@@ -157,17 +157,11 @@ impl View {
         let mut cursor_status = buffer.get_mark_display_coords(self.cursor).unwrap_or((0,0));
         cursor_status = (cursor_status.0 + 1, cursor_status.1 + 1);
         let status_text = format!("{} ({}, {})", buffer_status, cursor_status.0, cursor_status.1).into_bytes();
-        let status_text_len = status_text.len();
         let width = self.get_width();
         let height = self.get_height() - 1;
 
-
-        for index in 0..width {
-            let mut ch: char = ' ';
-            if index < status_text_len {
-                ch = status_text[index] as char;
-            }
-            self.uibuf.update_cell(index, height, ch, CharColor::Black, CharColor::Blue);
+        for (index, ch) in status_text.iter().enumerate().take(width) {
+            self.uibuf.update_cell(index, height, *ch as char, CharColor::Black, CharColor::Blue);         
         }
```

Comment:
If you build and run iota before and after this change, you'll see what I mean about the status bar. It goes full width before, and only as far as the status text after.